### PR TITLE
Updates home path to find_commodity globally

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

HOTT-<TODO>

### What?

I have added/removed/altered:

- [x] Altered references to sections_path to be browse_sections_path

### Why?

I am doing this because:

- Avoids unnecessary redirects
- Cleans up references to an unused path we might get rid of later

### CODE REVIEW

File: app/helpers/goods_nomenclature_helper.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def goods_nomenclature_path(path_opts = {})
 
     case current_goods_nomenclature_code&.size
     when nil
-      sections_path(path_opts)
+      find_commodity_path(path_opts)
```
Comment (jebw) [inline app/helpers/goods_nomenclature_helper.rb]: Not sure on this - should this one be going to `browse_sections_path`?

Comment (willfish) [reply app/helpers/goods_nomenclature_helper.rb]: So the fallback path here should be the home page, yeah, so this is consistent with previous behaviour.

Comment (willfish) [reply app/helpers/goods_nomenclature_helper.rb]: Unfortunately we can't use root_path

**Comment by willfish** [comment]
Add home_path helper to make it easier to replace essentially the root_path that is hijacked in production

**Review by jebw — APPROVED** [review]
LGTM

**Pull request merged**